src/main/plugin/iso19115-3.2018.che/loc/fre/codelists.xml

```diff
@@ -484,6 +485,75 @@
       </description>
     </entry>
   </codelist>
+
+  <codelist name="che:CHE_DS_AssociationTypeCode" alias="associationType">
```

Comment:
At this point, I am affraid it won't work:

gn expect codelist to use the name of the element it used in this case mri:DS_AssociationTypeCode. So if extra value have to be added, they have to be added in mri:DS_AssociationTypeCode.

The is already one L2695, with che specifics (in english), the standard one being erased from the beginning of the file... (erased later in this commit).

Simplier is maybe to use here the standard name even if specific.